test/source/tests/Test__timeUtils.bs

```diff
@@ -0,0 +1,168 @@
+' Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+' This product includes software developed at Datadog (https://www.datadoghq.com/).
+' Copyright 2022-Today Datadog, Inc.
+
+'----------------------------------------------------------------
+' Main setup function.
+' @return A configured TestSuite object.
+'----------------------------------------------------------------
+function TestSuite__TimeUtils() as object
+    this = BaseTestSuite()
+    this.Name = "TimeUtils"
+
+    this.addTest("WhenGetTimestamp_ThenReturnsTimestampInMilliseconds", TimeUtilsTest__WhenGetTimestamp_ThenReturnsTimestampInMilliseconds)
+
+    this.addTest("WhenNanosToMillis_ThenConvertsToMillis", TimeUtilsTest__WhenNanosToMillis_ThenConvertsToMillis)
+    this.addTest("WhenMillisToNanos_ThenConvertsToNanos", TimeUtilsTest__WhenMillisToNanos_ThenConvertsToNanos)
+    this.addTest("WhenMillisToSecs_ThenConvertsToSeconds", TimeUtilsTest__WhenMillisToSecs_ThenConvertsToSeconds)
+    this.addTest("WhenSecsToNanos_ThenConvertsToNanos", TimeUtilsTest__WhenSecsToNanos_ThenConvertsToNanos)
+    this.addTest("WhenSecsToMillis_ThenConvertsToMillis", TimeUtilsTest__WhenSecsToMillis_ThenConvertsToMillis)
+
+    return this
+end function
+
+'----------------------------------------------------------------
+'
+'----------------------------------------------------------------
+function TimeUtilsTest__WhenGetTimestamp_ThenReturnsTimestampInMilliseconds() as string
+    ' Given
+    ' hardcoded values to get a valid range
+    timestampMS_1jan2023 = 1672527600000
+    timestampMS_1jan2024 = 1704063600000
+
+    ' When
+    result = datadogroku_getTimestamp()
+
+    ' Then
+    Assert.that(result).isInRange(timestampMS_1jan2023, timestampMS_1jan2024)
```

Comment:
Are you going to have to update this test every year?

Reply:
Yep, that's the only way I could find to test this without duplicating the implementation in the test.